resources/languages/it/rules/time.clj

```diff
@@ -17,31 +17,31 @@
   ;;
 
   "named-day"
-  #"(?i)luned[ìi]|lun?\.?"
+  #"(?i)luned[iì]|lun\.?"
   (day-of-week 1)
 
   "named-day"
-  #"(?i)marted[iì]|mar?\.?"
```

Comment:
That's exactly the problem we were facing that triggered my change... add that two letter abbreviations for weekdays are rarely used